AWS SAM CLI instructions recommends invalid BuildProperties

The instructions at https://www.prisma.io/docs/guides/deployment/serverless/deploy-to-aws-lambda#loading-required-files say to add the following to each function's Metadata.BuildProperties:

```yml
Loader:
  - .prisma=file
  - .so.node=file
AssetNames: '[name]'
```

However, in so doing, sam build fails with 

```
Error: NodejsNpmEsbuildBuilder:EsbuildBundle - Esbuild Failed: ✘ [ERROR] Invalid option in build() call: "loader:"
```


and 


```
Error: NodejsNpmEsbuildBuilder:EsbuildBundle - Esbuild Failed: ✘ [ERROR] Invalid option in build() call: "asset-names:"
```

--- a/content/200-orm/200-prisma-client/500-deployment/201-serverless/400-deploy-to-aws-lambda.mdx
+++ b/content/200-orm/200-prisma-client/500-deployment/201-serverless/400-deploy-to-aws-lambda.mdx
@@ -101,17 +101,6 @@ if (process.env.NODE_ENV !== 'production') {
 }
 ```
 
-You will also need to define how to bundle these files with esbuild by adding the following lines to `Metadata.BuildProperties` in your `template.yaml`:
-
-```yaml file=template.yaml showLineNumbers
-Loader:
-  - .prisma=file
-  - .so.node=file
-AssetNames: '[name]'
-```
-
-This will make sure that files needed by Prisma ORM will be included in the AWS SAM build.
-
 ## Deploying with the Serverless Framework
 
 ### Loading environment variables via a `.env` file

